nifi-extension-bundles/nifi-standard-services/nifi-dbcp-service-bundle/nifi-dbcp-service/src/main/java/org/apache/nifi/dbcp/DBCPConnectionPool.java

```diff
@@ -272,4 +276,14 @@ protected Driver getDriver(final String driverName, final String url) {
             }
         }
     }
+
+    @OnRemoved
+    public void onRemove() {
```

Comment:
If we enable/disable the controller service many times, a new driver instance might be created and registered, no? Should we have a similar logic added in `@OnDisabled`?